src/topp/MzTabExporter.cpp

```diff
@@ -730,7 +730,7 @@ namespace OpenMS
 
       // currently we don't save how many channels we saved in a consensus feature so we scan the file
       // for the maximum max index in a consensus feature and take this as number of study variables
```

Comment:
```
  row.peptide_abundance_stdev_study_variable[study_variable];
```
Calls the default constructor for the entry which sets this entry to "null" aka not set.